bindings/ios/MEGASdk.h

```diff
@@ -5435,32 +5534,27 @@ typedef NS_ENUM(NSUInteger, StorageState) {
  */
 - (NSString *)fingerprintForFilePath:(NSString *)filePath;
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-
 /**
- * @brief Get a Base64-encoded fingerprint from an ALAssetRepresentation and a modification time
+ * @brief Get a Base64-encoded fingerprint from a NSData and a modification time
  *
  * If the input stream is nil, has a negative size or can't be read, this function returns nil
  *
- * @param assetRepresentation ALAssetRepresentation that provides the data to create the fingerprint
+ * @param data NSData that provides the data to create the fingerprint
  * @param modificationTime Modification time that will be taken into account for the creation of the fingerprint
  * @return Base64-encoded fingerprint
  */
-- (NSString *)fingerprintForAssetRepresentation:(ALAssetRepresentation *)assetRepresentation modificationTime:(NSDate *)modificationTime;
-
-#pragma clang diagnostic pop
+- (NSString *)fingerprintForData:(NSData *)data modificationTime:(NSDate *)modificationTime;
```

Comment:
We had `ALAssetRepresentation` because maybe someone wants to use our SDK to develop an application and support iOS 7. But it is not a problem at all if we remove this.

Reply:
I removed it is because the `ALAssetRepresentation` is already deprecated in iOS quite a time ago.